tac/member-info/election-timeline.md

```diff
@@ -0,0 +1,97 @@
+
+      3) electing annually a chairperson (the “TAC Chair”) to preside over meetings, set the agenda for meetings, ensure meeting minutes are taken and who will also serve on the Governing Board as the TAC’s representative (the “TAC Representative”);
+
+and
+
+      4) such other matters related to the technical role of the TAC as may be communicated to the TAC by the Governing Board.
+
+   2) The voting members of the TAC consist of:
+
+      1) six representative elected by the maintainers of the Technical Projects as a group; and
+
+      2) five representatives selected by the Governing Board.
+
+   3) TAC meetings are intended to be open for public observation.
+
+   4) The TAC Chair will preside over meetings, ensure minutes are taken and drive the TAC agenda with input from the TAC representatives.  The TAC may appoint a Vice Chair to preside over TAC meetings and ensure that minutes are taken when the Chair is not available.  A Vice Chair does not serve as the alternate to the TAC Representative with respect to the Governing Board.
+
+
+#### TAC Nominees
+
+##### Nomination period: 14 Oct 2024 - 31 Oct 2024 
+
+To nominate yourself for the TAC, first determine if you are eligible (see above). If you are eligible, [file an issue on GitHub](https://github.com/LF-Decentralized-Trust/governance/issues) with your nomination statement.
+
+##### Nomination Timeline
+
+* 14 Oct 2024 - The community architects will gather the GitHub IDs of the active contributors and update the eligibility check site.
+* 14 Oct 2024 - Update the email templates to reflect the dates for the 2024 election cycle.
+* 14 Oct 2024 - Publish the template for submitting nominations and open the call for nominees.
+* 14 Oct 2024 - Communication sent to LF Decentralized Trust Discord announcement channel announcing the call for TAC nominations using the above template.
+* 14 Oct 2024 - TAC will be notified via email that the call for nominations has begun.
+* 23 Oct 2024 - Reminder communication sent about nominations to LF Decentralized Trust Discord announcement channel about the call for TAC nominations using the above template.
+* 31 Oct 2024 - Nominations close EOD Pacific. Go through activity log of the election list during the nomination period to see if any nomination statements were tagged as spam and rejected.  Follow up with people as needed.
+* 31 Oct 2024 - All nominees must have created their nomination statement as a GitHub Issue on [LF-Decentralized-Trust governance](https://github.com/LF-Decentralized-Trust/governance/issues) repo prior to EOD.
+
+
+#### Election Process
+##### Voting
+
+###### Voting period: 01 Nov 2024 - 14 Nov 2024 
+
+The individuals eligible to vote in the TAC election is defined in §4.a.ii of the LF Decentralized Trust Charter. The election will be created using https://vote.heliosvoting.org/. The vote will elect 6 people from the list of TAC nominees.
```

Comment:
need to update section # here. I cannot find the entire LF Decentralized Trust charter. The one on lfdecentralizedtrust.org seems to only include section 1 or a portion of section 1.

Reply:
@tkuhrt it looks like this section doesn't exist in the new charter.

Reply:
I noticed that too. This is all I found:
> 5.b.i. six representative elected by the maintainers of the Technical Projects as a group;

From this I take that maintainers are the only ones eligible to vote. For some reason, we no longer define what maintainers are. Strange.

Reply:
I think someone in a proper group in the yaml file?

Reply:
I agree. “Maintainer” means elevated GitHub privileges, and that is all captured in the yaml file.  The only question I have is whether all the roles are “evevated” — e.g. READ and perhaps even TRIAGE may not be.  Or could be included, since just having a role implies elevated from a member of the general population that can submit a PR or an issue.

Are there any repos with individuals identified with roles, or have you cleaned all of those up in moving to the yaml file?

Reply:
@swcurran if it isn't in the YAML file, it doesn't exist :)